make/proto.mk

```diff
@@ -11,11 +11,14 @@ ifeq ($(UNAME_OS),Darwin)
 	CLANG_FORMAT_BIN ?= /usr/local/opt/clang-format/bin/clang-format
 endif
 
-proto-lint: $(BUF)
-	$(BUF) check lint --error-format=json
+proto-lint: 
+	$(DOCKER_BUF) check lint --error-format=json
+.PHONY: proto-lint
 
-proto-check-breaking: $(BUF)
-	$(BUF) check breaking --against-input '.git#branch=master'
+proto-check-breaking:
+	$(DOCKER_BUF) check breaking --against-input $(HTTPS_GIT)#branch=master
```

Comment:
i think HTTPS is unnecessary. we can compare it against local copy

Reply:
Actually, github workflow is failing when using just `.git`. Please check [this](https://github.com/ovrclk/akash/pull/803/checks?check_run_id=1224481285) once for reference

Reply:
looks like it needs actions/checkout to pull `master` too

Reply:
I already included `actions/checkout@master` in workflow. Will try to check with `.git` once again. But this PR can be merged as issue is with `proto.yml` workflow which is not yet included in master.

Reply:
see if [this works](https://github.com/ovrclk/akash/pull/803#discussion_r501853316)